api/handlers_images.go

```diff
@@ -92,3 +93,49 @@ func (s *server) ImageGetHandler(w http.ResponseWriter, r *http.Request) {
 
 	handleResponseOk(w, toEc2ImageResponse(out[0]))
 }
+
+func (s *server) UpdateImageTagsHandler(w http.ResponseWriter, r *http.Request) {
+	w = LogWriter{w}
+	vars := mux.Vars(r)
+	account := s.mapAccountNumber(vars["account"])
+	id := vars["id"]
+
+	role := fmt.Sprintf("arn:aws:iam::%s:role/%s", account, s.session.RoleName)
+	policy, err := tagCreatePolicy()
+	if err != nil {
+		handleError(w, err)
+		return
+	}
+
+	req := &Ec2UpdateTagRequest{}
+	if err := json.NewDecoder(r.Body).Decode(req); err != nil {
+		handleError(w, err)
+		return
+	}
```

Comment:
Also check that there are Tags in the input, e.g.
```
	if req.Tags == nil {
		handleError(w, apierror.New(apierror.ErrBadRequest, "missing required field: tags", nil))
		return
	}
```

Reply:
Done.